build.xml

```diff
@@ -173,15 +173,16 @@
     <copy todir="${classes.dir}/com/google/javascript/jscomp/js">
       <fileset dir="${src.dir}/com/google/javascript/jscomp/js" />
     </copy>
+
+    <!-- Zip the externs and move them to the classes folder -->
+    <zip destfile="${classes.dir}/externs.zip" basedir="${externs.dir}" includes="**/*.js" />
```

Comment:
No. Doing that didn't include the subfolders and files. I explicitly had to change it so that the externs folder structure was included/preserved.